src/components/Search/Filters/Control/DesktopControl.tsx

```diff
@@ -114,7 +114,7 @@ function DesktopControl({
           title={t("ageTitle")}
           hasSelection={!!filters.age}
           clear={() => {
-            const { age, ...updatedFilters } = filters;
+            const { ...updatedFilters } = filters;
```

Comment:
we need `age` here, b/c we want `updatedFilters` to represent only the filters that are not age - otherwise setFilters on line 118 doesn't actually do anything. is this b/c of the no unused vars rule? if so, we may just need to eslint-disable-next-line

Reply:
Good to know! Yes - this is because of that linting rule. I can disable that rule here an leave a note.

Reply:
actually, looks like there's an eslint option that accounts for this: https://eslint.org/docs/latest/rules/no-unused-vars#ignorerestsiblings

Reply:
Oop just saw this comment. I actually renamed the prop with an underscore, but this looks good too! I almost like the explicit plucking with the underscore renaming, but this rule does make this fall in line with expected JS behavior. I'll try adding it in.

Reply:
Placed the rule in the`package.json` file to account for plucking behavior.